src/components/pages/CustomerRegistration/CustomerRegistration.js

```diff
@@ -22,7 +23,7 @@ function RegisterCustomer() {
 
   const onSubmit = data => {
     axios
-      .post('https://labspt12-express-groomer-a-api.herokuapp.com/customers', {
+      .post(`${process.env.REACT_APP_API_URI}customers`, {
```

Comment:
there should be a trailing slash at the end of the env variable to keep things consistent

Reply:
done